uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/instances/sickness-leave-request-instance_test.js

```diff
@@ -114,5 +114,41 @@ define([
         expect(instance.required_documents).toEqual('');
       });
     });
+
+    describe('with required documents', function() {
```

Comment:
You copy-pasted these tests from the controller, but given that you moved them to an instance test suite, the test should be at least structured differently